scrapy/utils/display.py

```diff
@@ -3,20 +3,61 @@
 """
 
 import sys
+from distutils.version import LooseVersion as check
```

Comment:
`check` seems rather ambiguous, maybe `parse_version` or similar?

Reply:
Yes, I changed it to `parse` .
Thanks!

Reply:
`parse` still seems rather generic :smile:

Reply:
@Gallaecio 😄You re' right.
but to use `parse_version`, I have to use two if statements(like below) as single if statement is failing one flake test of crossing 79 characters limit in a single line.
```
    if sys.platform == "win32":
        if parse_version(version()) >= parse_version("10.0.14393"):
```

Reply:
We are not currently enforcing the 79-chars limit. In fact, we're working towards a higher limit (#4237). For the time being, please add this file to `pytest.ini` with the code `E501`.

Reply:
Alright! single `if` statement will be good then.
@elacuesta thanks for the info : )

Reply:
>  For the time being, please add this file to `pytest.ini` with the code `E501`.
Done 👍